enhancing/enhancer.js

```diff
@@ -38,6 +38,10 @@ function fail(item) {
   if (enhancement >= 15) durability -=10;
   if (enhancement > 16) enhancement -=1;
 
+  if (durability <= 0) {
+    return 'message: Item has been destroyed';
```

Comment:
aha, not a bad idea. I was more leaning in the direction of resetting any negative durability values back to zero (since having a negative value doesn't make sense), but sending a custom message to the user is also great